src/interfaces/navigationItem.ts

```diff
@@ -1,6 +1,9 @@
-export interface ISideBarItem {
+import { IconType } from "react-icons";
+
+export interface ISidebarItem {
```

Comment:
Maybe either the name of the file or the interface can be changed to that of the other to reach consistency :)